README.md

```diff
@@ -136,16 +164,11 @@ $ python app.py
 ```
 Alternatively, after running `rosgui` you can run `startgui` to run the GUI. If you run `startgui` it will also run the `updateEnv` alias which makes sure that `env.js` is setup.
 
-### ffmpeg
```

Comment:
Shouldn't this stay in the README? since this part explains how to setup manually, so that means it was not installed with the setup script

Reply:
I made sure it does install, since it used to be optional. The rosdep command before catkin_make installs all package dependencies, including ffmpeg